controllers/productController.js

```diff
@@ -38,9 +38,70 @@ const getDetailProducts = catchAsync(async (req, res) => {
   res.status(200).json({ data : detailProduct });
 });
 
+const createProduct = catchAsync(async(req, res) => {
+  const userId = req.user;
+  const {
+    name,
+    firstDate,
+    lastDate,
+    price,
+    description,
+    participants,
+    dicountRate,
+    scheduleTitle,
+    scheduleEtc,
+    classTypeId,
+    subCategoryId,
+    levelId,
+    locationName,
+    locationLatitude,
+    locationLongitude,
+    locationPlaceUrl,
+    locationGroupName,
+    schedules,
+  } = req.body;
+
+  const files = req.files;
+  const thumbnailImageUrl = files[0].location;
+  const productImagesUrl = [];
+  for (let i = 1; i < files.length; i++) {
+    productImagesUrl.push(files[i].location)
+  }
```

Comment:
productImagesUrl이라는 변수에 files 배열에 해당하는 값들을 얕은 복사를 다음과 같이 작성할 수 있을 것 같습니다~!
```suggestion
  const productImagesUrl = [...files]; 
```